gravitee-am-management-api/gravitee-am-management-api-rest/src/main/java/io/gravitee/am/management/handlers/management/api/authentication/filter/JWTAuthenticationFilter.java

```diff
@@ -81,6 +83,9 @@ public class JWTAuthenticationFilter extends AbstractAuthenticationProcessingFil
     @Value("${jwt.cookie-domain:}")
     private String jwtCookieDomain;
 
+    @Value("${http.blockingGet.timeoutMillis:120000}")
+    private long blockingGetTimeoutMillis = -1;
```

Comment:
Why are there 2 different defaults?

Reply:
it was to be sure that the default value in the annotation was used.
updated